### PR TITLE
msp430 spi speed fix + USIC/USCI cleanup

### PR DESCRIPTION
**Pull request by malosek**
Hello,

minor issue with spi speed configuration + definitive solution for USIC/USCI confusion.

wbr
malo

### CODE REVIEW

**Comment by haukepetersen** [comment]
ups :-) Looks good, untested ACK

**Comment by alignan** [comment]
:+1: ACK

File: cpu/msp430fxyz/periph/spi.c [hunk 1]
```diff
@@ -136,20 +142,26 @@ int spi_init_master(spi_t dev, spi_conf_t conf, spi_speed_t speed)
...
         default:
             /* other clock speeds are not supported */
             return -1;
     }
+
+    /* make sure the is not smaller then 2 */
+    if (br < 2) { 
+        br = 2;
+    }
+
```
Comment (LudwigKnuepfer) [inline cpu/msp430fxyz/periph/spi.c]:
```
cpu/msp430fxyz/periph/spi.c:92: trailing whitespace.
+ 
cpu/msp430fxyz/periph/spi.c:161: trailing whitespace.
+    if (br < 2) { 
cpu/msp430fxyz/periph/spi.c:164: trailing whitespace.
+ 
cpu/msp430fxyz/periph/spi.c:92: trailing whitespace.
+ 
cpu/msp430fxyz/periph/spi.c:161: trailing whitespace.
+    if (br < 2) { 
cpu/msp430fxyz/periph/spi.c:164: trailing whitespace.
+ 
ERROR: This change introduces new whitespace errors
```

**Comment by malosek** [comment]
Should be better now, noticed even bad alignment of break statement.
Sorry for the inconvenience.

wbr
malo

**Comment by haukepetersen** [comment]
CI is happy -> go. Thanks for the fix!

**Pull request merged**